### ISSUE TITLE
Flatpak build

### ISSUE THREAD

**Issue by LeandroStanger** [cdemu/cdemu#2]
Please, consider supporting [Flatpak](http://docs.flatpak.org/en/latest/introduction.html#reasons-to-use-flatpak) and publishing CDemu on [Flathub](https://github.com/flathub/flathub/wiki/App-Submission).

**Comment by hstokset** [comment]
TL;DR; IMO using Flatpak is pointless for systems software.

In my opinion there are times where it makes sense to use something like Flatpak and there are times where it does not.

Possible advantages can include things like packaging and containerization. All good things for a third-party application like for instance games, where you want to protect the system from bugs in the software, make installation easier by bundling all the libraries and so on.

But like most things there are also downsides, bundling libraries also means the software blow up in size, which might be a reasonable trade-off in some circumstances. Containerization is also pointless for systems software which per definition requires access to your system. Packaging will arguably be worse since we can't integrate as well with the rest of the system and we already support the major packaging options.
